src/Integration.Vsix/Protobuf/.gitignore

```diff
@@ -0,0 +1 @@
+*.cs
```

Comment:
This is how it was done in `sonar-csharp` too. And I think it's better this way, because this rule is really local to this directory, so good to encapsulate it here, the top level doesn't need to know.